manifests/charts/dash/values.yaml

```diff
@@ -12,7 +12,7 @@ image:
 kubesec:
   registry: m9sweeper
   repository: kubesec
-  tag: latest
+  tag: 2.12.0
```

Comment:
Also need to change registry do docker.io and repository to kubesec/kubesec, I think

Reply:
So that we are loading from dockerhub

Reply:
https://hub.docker.com/r/kubesec/kubesec/tags

Reply:
We can also remove this file - https://github.com/m9sweeper/m9sweeper/blob/main/dash/kubesec/Dockerfile